app/models/testimony.rb

```diff
@@ -1,7 +1,8 @@
 class Testimony < ApplicationRecord
-  has_one_attached :photo
+  has_one :photo, as: :imageable, dependent: :destroy, class_name: "Image"
+  accepts_nested_attributes_for :photo, reject_if: :all_blank, allow_destroy: true
```

Comment:
need to make photo optional, if it is not possible, probably can just remove the photo for now

Reply:
Should have been answered by my comment above